### PR TITLE
Expand the readme to add instructions on accessing via nested vms.

### PR DESCRIPTION
**Pull request by hatched**
## Done

Expand the readme to add instructions on accessing via nested vms.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,14 +14,57 @@ Assuming you already have [Docker](https://www.docker.com/) installed, you can s
...
+./run exec yarn run generate-release-tarball
+```
+
+### Accessing a lxd bootstrapped Dashboard that's also within a `multipass` VM
+
+When you want to access a dashboard that was automatically installed with a `juju bootstrap` within an lxd in a `multipass` vm from the host.
+
+- [Create a ssh tunnel](#creating-a-tunnel-from-an-lxd-controller-to-a-multipass-host)
+- In the host
+  - `multipass info dev | grep IPv4` and take note of the IP address.
```
Comment (clagom) [inline README.md:39]: Can we specify 'dev' is the name of your Multipass vm?

File: README.md [hunk 2]
```diff
@@ -12,14 +14,57 @@ Assuming you already have [Docker](https://www.docker.com/) installed, you can s
...
+  - In your browser visit `http://<that ip>:17070`
```
Comment (clagom) [inline README.md:40]: got a 'This site can’t be reached' refused to connect. My multipass vm was running properly when I tried

File: README.md (hunk 1 above)
Comment (clagom) [reply README.md:39]: meaning, that users need to change it with their own vm name

File: README.md [hunk 3]
```diff
@@ -12,14 +14,57 @@ Assuming you already have [Docker](https://www.docker.com/) installed, you can s
...
+    - `isJuju` to `true`.
+
+### Creating a tunnel from an lxd controller to a `multipass` host.
+
+If you're bootstrapping a Juju controller within a `multipass` VM you will not have access to the dashboard from the host because there is no network path to it. To create a tunnel to access the dashboard follow these steps.
+
+- After bootstrapping a controller on `lxd` in your `multipass` VM...
+- In the multipass vm
+  - `juju switch controller && juju ssh 0`
+- In the lxd container
```
Comment (clagom) [inline README.md:62]: How do I access the LXD container from my Multipass?

File: README.md (hunk 1 above)
Comment (hatched) [reply README.md:39]: Good catch, done.

File: README.md (hunk 3 above)
Comment (hatched) [reply README.md:62]: the `juju ssh 0` in the above command will ssh you into the lxc that Juju created for it's "machine 0"

**Comment by hatched** [comment]
This will be easier for you to follow @clagom so I'll merge it and then you can read it without all the diff cruft. 👍

**Pull request merged**